vital/types/metadata.h

```diff
@@ -395,11 +420,12 @@ class VITAL_EXPORT metadata
     using ptr_t = std::unique_ptr< result_t >;
     this->add( ptr_t{ new result_t{ tag_t::name(), data } } );
   }
+  //@}
```

Comment:
I can't Suggest this properly, but also:

```c++
  //@}

  /**
   * \brief  Add metadata item to collection.
   *
   * This method creates a new metadata item and adds it to the collection.
   * Unlike ::add, this method must be used if the \p data is an ::any, as
   * overload resolution will fail otherwise.
   *
   * \tparam Tag Metadata tag value.
   * \param data Metadata value.
   */
```

...and then (because it is no longer needed):

```suggestion
```